dashboard/utils.py

```diff
@@ -29,8 +29,8 @@ class MMTrack:
     enrollments = None
```

Comment:
**NOTE: ** ALL of the changes in this file, and the associated test file (dashboard/utils_test.py), have to do with renaming variables called `*course_id` to `*course_run_id`. The changes don't need to be reviewed in detail